### DISCUSSION TITLE
Use datetime-local for input in DateTime interface

### DISCUSSION THREAD

**Discussion by christianrr** [directus/directus#2755] [Feature Requests]
The html input element with its type datetime-local provides an usable datetime picker on most browsers (Chrome, Opera, Edge as well as most mobile browsers).

I think it would be a good idea to use it as input for the DateTime interface, which supports AFAIK only text input at the moment.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local

**Comment by rijkvanzanten** [comment]
We used to have that before actually. The differences between browsers, and the inability to style the picker and/or adjust the styling was a pretty big buzzkill.. 

I remember there was an actual blocker issue that made us switch it out, but I don't remember out of the top of my head what that was

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
